scalafix-tests/unit/src/test/scala/scalafix/tests/core/DisableRangeTest.scala

```diff
@@ -0,0 +1,65 @@
+package scalafix.tests.core
```

Comment:
Should I leave unit tests or scalafix-testkits are enough?

Reply:
testkit is enough I think, that avoids us testing against the scalafix internal API.